src/main/java/tictactoe/BoardConverter.java

```diff
@@ -11,38 +13,87 @@ public String toString(Marks[][] board) {
 
         for (Marks[] columns : board) {
             for (Marks mark : columns) {
-                str.append(createLine(idSpot, mark));
+                str.append(createSpot(idSpot, mark));
```

Comment:
What's represented by`idSpot` and `idLine`? I think the names do not really reveal the intent behind those variables.

Oh and as a side note `StringBuilder` is deprecated for a long time. There's no need to use one anymore. It's totally fine to just concatenate strings in Java.